integrations/dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/RumConfigurationExt.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.compose
+
+import com.datadog.android.rum.RumConfiguration
+import com.datadog.android.rum._RumInternalProxy
+import com.datadog.android.rum.tracking.ActionTrackingStrategy
+
+/**
+ * Enable [ActionTrackingStrategy] for Jetpack Compose automatic actions tracking, such as
```

Comment:
probably we should remove mention of `ActionTrackingStrategy` from this public doc, because it is not something that customer may set in the public API and it is not supposed to be used directly.